### DISCUSSION TITLE
Smoothing out load for artillery (playwright)

### DISCUSSION THREAD

**Discussion by zoe-jobson-IL** [artilleryio/artillery#2576] [Artillery]
I would like to achieve a load which smoothly rises from 1 to 75 over a given period of time (this is per fargate pod - in a recent test run we had 67 of these to bring the eventual load to >5k), I don't want just starting smoothly over time I want it to be actual users, so far the only fix I have discovered is to have 75 separate phases each a given period in length and each 1 user more than the one before
e.g.

```
  - duration: 1
    arrivalRate: 1
    maxVusers: 1
    name: user1
  - duration: 1
    arrivalRate: 1
    maxVusers: 2
    name: user2

```
 other solutions either result in chunky load (too many users being added at the same time) or loss of users - i.e. if one fails it doesn't get replaced by another. If we could specify an arrival rate of <1 per second it would help because with >50 pods we end up with 50 users added per second which is too much for our system to cope with, but experience has shown that for some of our tests the maximum number of users a fargate pod can handle is around 75.

**Comment by zoe-jobson-IL** [comment]
As an addendum to this, with the new cloud UI my load tests with 75 mini phases cause the cloud UI to fail, e.g. https://app.artillery.io/load-tests/tzr3p_88ecc9kt8xxx9dnccfweccprh7fgg_88bh

**Reply by manfromanotherland** [reply]
Hi @zoe-jobson-IL, Ed from Artillery here 👋

Can you share that test via the Share functionality (screenshot) or share a screenshot please?

Thanks!

![Share test](https://github.com/artilleryio/artillery/assets/213251/ab003b79-138c-4a14-a57a-a2703f53ba7e)

**Reply by zoe-jobson-IL** [reply]
No, when I try to open it enough to do so I get
![image](https://github.com/artilleryio/artillery/assets/72204483/58c63674-b195-49b4-82b4-d97d6e73442e)
You can see the share button is disabled

**Reply by manfromanotherland** [reply]
Ah bummer, thanks for trying. Taking a look now!

**Reply by manfromanotherland** [reply]
@zoe-jobson-IL The team just pushed a fix for the page load issue, could you try again?

**Reply by zoe-jobson-IL** [reply]
much better, thanks :D

**Reply by zoe-jobson-IL** [reply]
The reports tab is still playing up https://app.artillery.io/share/sh_c4d5e2a3921acfb89446e80e1bf509a17259022eee4f4cf5c5e6ab16fc1175c8
when you try to scroll down to lower charts it occasionally gives me a page unresponsive message
